package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "@rails/webpacker": "^5.2.1",
     "accessible-autocomplete": "^2.0.3",
+    "axios": "^0.21.1",
```

Comment:
Is this required? It's a pretty large library and only being used for a simple request.

Fetch when polyfilled with https://github.com/github/fetch covers our browser support and is considerably smaller :)

Reply:
We can change it later on - for now it does the work

Reply:
Is there a specific reason it can't be done now such as the work having a tight deadline? Or do you have a plan to remove the library once this PR is merged?

Reply:
Given today is sprint planning

Reply:
Done